tests/integration/test_fim/test_synchronization/test_registry_responses_win32.py

```diff
@@ -56,36 +56,33 @@
 '''
 import os
 import pytest
+from wazuh_testing import DATA, WAZUH_SERVICES_START
 from wazuh_testing.fim import (generate_params, create_registry, modify_registry_value, registry_parser,
                                KEY_WOW64_64KEY, REG_SZ)
 from wazuh_testing.tools.configuration import load_wazuh_configurations
-from wazuh_testing.tools.monitoring import FileMonitor
 from wazuh_testing.tools.services import control_service
 from wazuh_testing.fim_module.fim_synchronization import find_value_in_event_list, get_sync_msgs
-from wazuh_testing.fim_module.fim_variables import (SCHEDULE_MODE, WINDOWS_REGISTRY, SYNC_INTERVAL, SYNC_INTERVAL_VALUE,
-                                                    YAML_CONF_REGISTRY_RESPONSE, WINDOWS_HKEY_LOCAL_MACHINE,
-                                                    MONITORED_KEY)
-from wazuh_testing.wazuh_variables import DATA, WAZUH_SERVICES_START, WINDOWS_DEBUG, VERBOSE_DEBUG_OUTPUT
+from wazuh_testing.fim_module.fim_variables import (SCHEDULE_MODE, WINDOWS_REGISTRY, SYNC_INTERVAL,
+                                                    SYNC_INTERVAL_VALUE, YAML_CONF_REGISTRY_RESPONSE,
+                                                    WINDOWS_HKEY_LOCAL_MACHINE, MONITORED_KEY)
```

Comment:
Why do you use `wazuh_testing.fim_module.fim_variables`, and  `wazuh_testing.fim_module.fim_synchronization` when we try to use `wazuh_testing.modules.fim`?

Reply:
The `module_fim` folder refactor has not been merged into master yet. It has been done in PR https://github.com/wazuh/wazuh-qa/pull/2728/files. Doing it again would require a lengthy amount of time, and the fix will be applied when both PRs are merged together in master.